.github/workflows/docker-build.yml

```diff
@@ -3,6 +3,12 @@ name: Build in Docker
 on:
   workflow_dispatch:
   workflow_call:
+    inputs:
+      tt-mlir-version:
+        type: string
+        description: 'Version of tt-mlir to use'
```

Comment:
Can you outline that it can be a branch, SHA, tag, etc?

Reply:
👍